python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -186,11 +192,6 @@ def __init__(self, *args, **kwargs):
                           help="Mode of delivering the files to their destination")
         self.add_argument("--create", action="store_true", help="Create new registry?")
         self.add_argument("--validity", type=int, required=True, help="Calibration validity period (days)")
-        self.add_argument("--calibType", type=str, default=None,
-                          choices=[None, "bias", "dark", "flat", "fringe", "sky", "defect"],
-                          help="Type of the calibration data to be ingested;"
-                               " if omitted, the type is determined from"
-                               " the file header information")
```

Comment:
If calibType can no longer be specified on the command line, I am pretty sure decam fringe ingestion will break, although it is already pretty broken (it must presently be run with mode=skip and the files must be moved/copied/linked manually to the calib repo).